supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -404,7 +482,7 @@ Do not use "BIOS" as a generic term to refer to computer firmware. Use "firmware
 // Ceph: General; kept as is
 [[bucket]]
 ==== image:images/yes.png[yes] bucket (noun)
-*Description*: 1) A _bucket_ in the S3 API contains objects. A bucket also defines access control lists (ACLs). Unlike folders or directories, buckets cannot contain other buckets. A bucket in the S3 API is synonymous with a _container_ in the Swift API. 2) The term "bucket" is also sometimes used in the context of a _CRUSH hierarchy_, but CRUSH buckets and S3 buckets are mutually exclusive concepts.
+*Description*: (1) A _bucket_ in the S3 API contains objects. A bucket also defines access control lists (ACLs). Unlike folders or directories, buckets cannot contain other buckets. A bucket in the S3 API is synonymous with a _container_ in the Swift API. (2) The term "bucket" is also sometimes used in the context of a _CRUSH hierarchy_, but CRUSH buckets and S3 buckets are mutually exclusive concepts.
```

Comment:
SE > Amazon Simple Storage Service (Amazon S3)

Reply:
This entry was already here, I am just fixing the (1) and (2) definition syntax.